### PR TITLE
12.0 afc 1486

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**

### CODE REVIEW

File: outplacement/models/outplacement.py [hunk 1]
```diff
@@ -56,6 +56,18 @@ def _read_group_employee_ids(self, employees, domain, order):
     meeting_remote = fields.Selection(selection=[('no','On Premice'),('yes','Remote')],string='Meeting type')
     uniq_ref = fields.Char(string='Uniq Id', size=64, trim=True, )
 
+    status = fields.Selection(string="status", selection=[
```
Comment (anderswallenquist) [inline outplacement/models/outplacement.py]: Use outplacement.state for this

File: outplacement/__manifest__.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
     'author': 'Vertel AB',
     'license': 'AGPL-3',
     'website': 'http://www.vertel.se',
-    'depends': ['base', 'hr', 'mail'],
+    'depends': ['base', 'hr', 'mail', 'partner_firstname'],
```
Comment (anderswallenquist) [inline outplacement/__manifest__.py]: I'm not sure that we need this depends here. You have to add first_name and last_name manually with relations to res partner

**Review by anderswallenquist — CHANGES_REQUESTED** [review]

File: outplacement_completion_report_ipf_client/models/client_config.py [hunk 3]
```diff
@@ -217,22 +179,25 @@ def post_request(self, outplacement, res_joint_planning_af_recordset):
...
             "genomforande_referens": outplacement.order_id.origin,
             "ordernummer": outplacement.order_id.name,
-            "personnr": outplacement.partner_id.company_registry,
+            "personnr": outplacement.partner_id.social_sec_nr,
             "unikt_id": unikt_id,
             "deltagare": {
                 "fornamn": outplacement.partner_id.firstname,
                 "efternamn": outplacement.partner_id.lastname,
-                "deltog_per_distans": outplacement.meeting_remote
+                "deltog_per_distans": "yes" if outplacement.meeting_remote else "no"
```
Comment (anderswallenquist) [inline outplacement_completion_report_ipf_client/models/client_config.py]: Meeting remote is "yes" or "no" as it is, not a Boolean

**Review by anderswallenquist — CHANGES_REQUESTED** [review]

File: outplacement/__manifest__.py (hunk 2 above)
Comment (SebastianGieser-Vertel) [reply outplacement/__manifest__.py]: I wasn't sure where to put it, but it's necessary for the completion report API and probably others as well so I thought it'd make sense to be here.

File: outplacement/models/outplacement.py (hunk 1 above)
Comment (SebastianGieser-Vertel) [reply outplacement/models/outplacement.py]: I can't find outplacement.state, did you mean outplacement.stage_id?

Comment (nils-vertel) [reply outplacement/models/outplacement.py]: Yes

File: outplacement/__manifest__.py (hunk 2 above)
Comment (nils-vertel) [reply outplacement/__manifest__.py]: If completion report API needs this, can this be put in the manifest of completion report?

Comment (SebastianGieser-Vertel) [reply outplacement/__manifest__.py]: I guess, just seems a bit strange to have it as a dependency in every api module that happens to need it.

File: outplacement_final_report_ipf_client/models/client_config.py [hunk 4]
```diff
@@ -229,7 +229,7 @@ def post_request(self, outplacement):
             "unikt_id": outplacement.uniq_ref,
             "inskickad_datum": str(outplacement.jp_sent_date),
             "rapportering_datum": outplacement.report_date, 
-            "status": outplacement.status,
+            "status": outplacement.stage_id.sequence,
```
Comment (anderswallenquist) [inline outplacement_final_report_ipf_client/models/client_config.py:232]: Is it sequence number?

**Review by anderswallenquist — APPROVED** [review]

File: outplacement_final_report_ipf_client/models/client_config.py (hunk 4 above)
Comment (SebastianGieser-Vertel) [reply outplacement_final_report_ipf_client/models/client_config.py:232]: I think but the number is the number I need

**Pull request merged**